activerecord/lib/active_record/railtie.rb

```diff
@@ -458,5 +458,13 @@ class Railtie < Rails::Railtie # :nodoc:
         end
       end
     end
+
+    initializer "active_record.attributes_for_inspect" do |app|
+      ActiveSupport.on_load(:active_record) do
+        if app.config.consider_all_requests_local
+          ActiveRecord::Base.attributes_for_inspect = :all
```

Comment:
I don't think so, since it is run in the `:active_record` load hook, it will be set before `Base` is inherited from. I tested in my app and it worked as expected. But I will add a test like you suggested in your other comment.

Reply:
What I mean is setting `config.active_record.attributes_for_inspect = [:uuid]`

Reply:
Ah sorry, wasn't thinking of that angle. Fixed it and added a test. Thanks!